staff/pablo-rey/cv/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CV Pablo Javier Rey Sobral</title>
+  </head>
+  <body>
+    <header>
+      <h1>Pablo Javier Rey Sobral</h1>
+      <h2>Curriculum Vitae</h2>
+    </header>
+    <main>
+      <section>
+        <h4>Personal information</h4>
+        <img
+          src="https://avatars1.githubusercontent.com/u/45286922?s=460&v=4"
+          alt="Pablo Javier Rey Sobral"
+        />
+        <time datetime="1975-07-10">Birthdate: 10/07/1975</time>
+        <p>Birthplace: Montevideo (Uruguay)</p>
+        <address>
+          <p>Carrer de Llull, 180 4º 1ª
+          <br />08005 Barcelona</p>
+          <a href="tel:+34607168306">+34 607168306</a>
+          <a href="mailto:pabloreysobral@gmail.com">pabloreysobral@gmail.com</a>
+        </address>
+        <a href="https://github.com/paulusrex">GitHub</a>
+      </section>
+      <section>
+        <h4>Academic information</h4>
+        <ol>
+          <li>
+            FP Especialista Informatica de Gestión (Not finished)
+            <time datetime="1995">1995</time>-<time datetime="1996">1996</time>
+          </li>
+          <li>
+            <a href="https://www.skylabcoders.com/">Skylab Coders Academy</a>
+            (In progress) [<time datetime="2019">2019</time>]
+          </li>
+        </ol>
+      </section>
+      <section>
+        <h4>Jobs</h4>
+        <ol>
+          <li>
+            <time>1998-2017</time>
```

Comment:
Like at line [36](https://github.com/manuelbarzi/skylab-bootcamp-201904/blob/8fa5511b50c63837d84b926666e1ab63312dc151/staff/pablo-rey/cv/index.html#L36) from your code, you can't indicate date ranges. The current specification has this limitation in what you can define with the <time> element.

Good job!